cms/static/js/models/xblock_info.js

```diff
@@ -186,10 +190,50 @@ function(Backbone, _, str, ModuleUtils) {
         },
 
         /**
+         * Return true if subsection content is required to show.
+         * currently, check subsection for an entrance exam.
+        */
+        showSubSectionContent: function() {
+           var type = this.getOverrideType();
+
+           if(type !=null) {
+               //hide the subsection if block is an entrance exam.
+               if (_.has(type, 'is_entrance_exam_subsection') && type['is_entrance_exam_subsection']) {
+                   return false;
+               }
+           }
+           return true;
+       },
+
+        /**
+         * Return the type of xblock.
+        */
+       getOverrideType: function() {
+           var type = null;
+           if(this.get('override_type') != null) {
+               var type = this.get('override_type');
+           }
+           return type;
+        },
+
+       getExamScore: function() {
```

Comment:
@cahrens -- is there a better-suited location already in place for this logic, or would it be better to add a new file/script?